src/Symfony/Component/Notifier/Bridge/LinkedIn/LinkedInOptions.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\LinkedIn;
+
+use Symfony\Component\Notifier\Bridge\LinkedIn\Share\AuthorShare;
+use Symfony\Component\Notifier\Bridge\LinkedIn\Share\LifecycleStateShare;
+use Symfony\Component\Notifier\Bridge\LinkedIn\Share\ShareContentShare;
+use Symfony\Component\Notifier\Bridge\LinkedIn\Share\VisibilityShare;
+use Symfony\Component\Notifier\Message\MessageOptionsInterface;
+use Symfony\Component\Notifier\Notification\Notification;
+
+/**
+ * @author Smaïne Milianni <contact@smaine.me>
+ *
+ * @experimental in 5.2
+ */
+final class LinkedInOptions implements MessageOptionsInterface
+{
+    private $options = [];
+
+    public function __construct(array $options = [])
+    {
+        $this->options = $options;
+    }
+
+    public function toArray(): array
+    {
+        return $this->options;
+    }
+
+    public function getRecipientId(): ?string
+    {
+    }
+
+    public static function fromNotification(Notification $notification): self
+    {
+        $options = new self();
+        $options->specificContent((new ShareContentShare($notification->getSubject())));
+
+        if ($notification->getContent()) {
+            $options->specificContent((new ShareContentShare($notification->getContent())));
+        }
+
+        $options->visibility((new VisibilityShare()));
+        $options->lifecycleState((new LifecycleStateShare()));
+
+        return $options;
+    }
+
+    public function contentCertificationRecord(string $contentCertificationRecord): self
```

Comment:
This setter and others from below don't have verb at the beginning of a name to describe their actions.
`setContentCertificationRecord`

Reply:
inspired from https://github.com/symfony/slack-notifier/blob/master/Block/SlackActionsBlock.php#L27

Reply:
I think this is not better name of method. IMHO.